pkgs/development/libraries/wayland/protocols.nix

```diff
@@ -45,6 +45,9 @@ stdenv.mkDerivation (finalAttrs: {
     pkgConfigModules = [ "wayland-protocols" ];
   };
 
+  passthru.updateScript = gitUpdater {
```

Comment:
Shouldn't meta come after passthru? Also we could combine those 3 passthrus into one.